src/main/java/org/veupathdb/service/eda/ds/utils/CommonFormats.java

```diff
@@ -0,0 +1,16 @@
+package org.veupathdb.service.eda.ds.utils;
+
+import java.time.LocalDate;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+
+public class CommonFormats {
+  public static final DateTimeFormatter TABULAR_DATE_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd");
```

Comment:
I thought we used to output date cols as datatimes (yyyy-MM-ddThh:mm:ss).  Maybe we don't do this any more?

Reply:
Can of worms there!

The client hasn't been updated to handle bubble dates yet, so the best thing is for the back end to do the most consistent thing and we will take care of whichever format (we'll likely not display any time component to the user, but no problem handling the full date-time in the response and subsequent calculations)